book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  # attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_book_to_rentals(book)
+    @rentals.push(book)
+    book.rentals = self
+  end
```

Comment:
- [ ] @danielamoreno699 great work implementing the changes requested, however, you did it wrongly, as the previous reviewer requested you add the `add_rental` method that takes in the `person, date, and book itself`, this enables us to achieve the 'has-many' association` between the person and book classes.